Source/Dafny/ConcreteSyntax/ConcreteSyntaxTree.cs

```diff
@@ -6,10 +6,14 @@
 using JetBrains.Annotations;
 
 namespace Microsoft.Dafny {
-  public enum BraceStyle { Nothing, Space, Newline }
+  public enum BraceStyle {
+    Nothing,
+    Space,
+    Newline,
+    Pindent
```

Comment:
I don't like the name `Pindent`. Does `P` refer to `Python` ? I'd prefer to keep `ConcreteSyntaxTree` as language agnostic as possible. How about the name `NoBrace` instead of `Pindent` ?

Reply:
I would rather replace "PIndent" by "SpaceIndent" to keep being language agnosticism, but being clear about what delimits the code. In other languages, NoBrace could also encompass `EndIf`, or sometimes there is no ending block marker at all (assembly). Let's do `SpaceIndent` or `Indent`.

Reply:
I prefer `NoBrace`, to make it clear that it's not just "indentation only", there also aren't any braces. I might even prefer `ColonNoBrace` to make it clear that the block starts with `:`.

(As a future refactoring it would probably be better to rename this enum to `BlockStyle` instead, but I don't want to make this PR larger)

Reply:
If I only look at the code, it's indeed not about indentation, but also not about colon either (the colon seems to be not part of the brace)
In that case, the appropriate keyword would be `NewLineNoBrace`.

Reply:
I guess "New line" is part of other  BraceStyles too but not mentioned in name explicitly, keeping that in mind `ColonNoBrace`  works better.

Reply:
Ah you're right, since the `):` tokens are added separately from the "brace", I agree that `NewlineNoBrace` is better.